prometric/src/summary/simple.rs

```diff
@@ -0,0 +1,63 @@
+//! Simple summary implementation
+//!
+//! Uses [`metrics_util::storage::Summary`] for the undelying representation
+
+use metrics_util::storage::Summary as Inner;
+
+use crate::summary::traits::{NonConcurrentSummaryProvider, Summary};
+
+/// A simple Summary metric implementation
+#[derive(Debug, Clone)]
+pub struct SimpleSummary {
```

Comment:
What does "Simple" mean here? Overall the distinction between rolling and simple isn't super clear, I have a rough idea but I want to be sure.